prusti-contracts-internal/src/lib.rs

```diff
@@ -57,3 +57,13 @@ pub fn extern_spec(attr: TokenStream, tokens: TokenStream) -> TokenStream {
 pub fn predicate(tokens: TokenStream) -> TokenStream {
     prusti_specs::predicate(tokens.into()).into()
 }
+
+#[proc_macro_attribute]
+pub fn export_spec(attr: TokenStream, tokens: TokenStream) -> TokenStream {
+    prusti_specs::export_spec(attr.into(), tokens.into()).into()
```

Comment:
Why the `into`s? The types should work out:
```suggestion
    prusti_specs::export_spec(attr, tokens)
```
Also in `prusti_use` below.